src/management/Akka.Management/Akka/Predef.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Akka
+{
+    public static class Predef
+    {
+        /// <summary>
+        /// Identity function to conform with the JVM API
+        /// </summary>
+        public static T Identity<T>(T x) => x;
```

Comment:
This should be part of `Akka` itself

Reply:
What does this do?

Reply:
@Aaronontheweb this is just an [identity function](https://stackoverflow.com/questions/1797502/is-there-a-scala-identity-function)